tests/e2e_tests/integration/01-login.spec.ts

```diff
@@ -34,16 +36,9 @@ test.describe('Login', () => {
       await page.fill(selectors.password, password);
       await page.click(`:is(button:has-text('Log in'))`);
       // confirm we have logged in successfully
-      await page.waitForSelector('text=License information');
+      await page.waitForSelector(loggedInText);
       await page.evaluate(() => localStorage.setItem(`onboardingComplete`, 'true'));
       await context.storageState({ path: 'storage.json' });
-      // now we can log out
-      await page.click('.header-dropdown', { force: true });
-      await page.click(`text=Log out`, { force: true });
-      const loginButton = page.getByRole('button', { name: /log in/i });
-      await loginButton.waitFor({ timeout: 7000 });
-      const loginVisible = await loginButton.isVisible();
-      expect(loginVisible).toBeTruthy();
```

Comment:
occasionally what happens to me is: I have a stale JWT in the local store, and there is no other standard way to exit this situation than pressing logout. I would ensure that we are logged out before we login, or even before we start any test case that requires or assumes a logged out user.

Reply:
that's what the `page` fixture ensures - a new test context, without any cookies etc. present - also we wouldn't be able to focus/ fill the email & password inputs otherwise 👌